src/main_app/components/NavBar/NavBar.js

```diff
@@ -32,6 +32,14 @@ function NavBar ({ variant, variantAtScrollTop, show = true, pathLogo = '/#top',
     return variant
   }, [atScrollTop, variant, variantAtScrollTop])
 
+  variant = useMemo(() => {
+    const isLight = variant?.find(element => element === 'light')
+
+    if (isLight) return variant.concat('white')
+
+    return variant
+  }, [variant])
```

Comment:
- No uses `find` y `concat` para trabajar con variants. Usá las utilidades que hablamos el viernes: `isVariant` y `combineVariants`.
- No uses las variantes del NavBar para el logo. Sí tenés que generar la variant del logo a partir de la del NavBar:
```suggestion
  const logoVariant = useMemo(() => {
    if (isVariant(variant, 'light')) return 'white'
    else return 'color'
  }, [variant])
```